cypress/e2e/pageObjects/OrangeHRMAddEmployee/dataUtils.ts

```diff
@@ -30,20 +29,17 @@ class api{
             method: 'GET',
             url: 'https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/pim/employees/'+empNumber+'/personal-details',
           }).then((response) => {
-            expect(response.status).to.equal(200);
             expect(response.body.data.employeeId).to.equal(employeeId);
           });
     }
 
-    DeleteEmployee(deleteEmployee: DeleteAPIBody){
+    DeleteEmployee(DeleteAPIBody: DeleteAPIBody){
```

Comment:
update the function as we explained (first get the employee by id then delete it )